pkg/ipfailover/keepalived/generator.go

```diff
@@ -144,9 +155,10 @@ func GenerateDeploymentConfig(name string, options *ipfailover.IPFailoverConfigC
 	podTemplate := &kapi.PodTemplateSpec{
 		ObjectMeta: kapi.ObjectMeta{Labels: selector},
 		Spec: kapi.PodSpec{
-			HostNetwork: true,
-			Containers:  containers,
-			Volumes:     generateVolumeConfig(),
+			HostNetwork:  true,
+			NodeSelector: generateNodeSelector(name, selector),
```

Comment:
Do we need a different flag name if --label on router and ipfailover commands means different things?